src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,379 @@
+            break;
+        case ('E'):
+            int eventIndex = fileLine.indexOf("(at:");
+            description = fileLine.substring(NEW_TASK_INDEX, eventIndex);
+            String event = getTimeFromFile(fileLine, eventIndex);
+            addNewEvent("event" + description + "/at" + event);
+            break;
+        }
+    }
+
+    private static String getTimeFromFile(String fileLine, int timeIndex) {
+        return fileLine.substring(timeIndex + 4, fileLine.length() - 1);
+    }
+
+    private static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(textToAdd);
+        fw.close();
+    }
+
+    private static String getAllTaskListData() {
+        String data = "";
+        for (int i = 0; i < maxTaskIndex; i++) {
+            data = data + (tasks.get(i).toString()) + "\n";
+        }
+        return data;
+    }
+
+    private static void enumerateTasks() {
+        int currentTaskIndex;
+        for (int i = 0; i < maxTaskIndex; i++) {
+            currentTaskIndex = i + 1;
+            System.out.println(currentTaskIndex + "." + tasks.get(i).toString());
+        }
+    }
+
+    private static void listBeginMessage() {
+        System.out.println("Here are the tasks in your list:");
+    }
+
+    private static void deleteTask(String input) {
+        int taskNumberToDelete = (Integer.parseInt(input.substring(DELETE_START).strip()) - 1);
+        deletedTaskMessage(taskNumberToDelete);
+        tasks.remove(taskNumberToDelete);
+    }
+
+    private static void deletedTaskMessage(int deletedTaskIndex) {
+        System.out.println("Noted. I've removed this task: ");
+        System.out.println(tasks.get(deletedTaskIndex));
+        System.out.println("Now you have " + (maxTaskIndex - 1) + " tasks in the list." + "\n");
+    }
+
+    private static void verifyValidInput(String input, Command c) throws EmptyInputException, InvalidEventTimeException, InvalidDeadlineTimeException {
+        if (isEmptyInput(input, c)) {
+            throw new EmptyInputException();
+        }
+
+        switch (c) {
+        case EVENT:
+            if (validEventTime(input)) {
+                break;
+            } else {
+                throw new InvalidEventTimeException();
+            }
+        case DEADLINE:
+            if (validDeadlineTime(input)) {
+                break;
+            } else {
+                throw new InvalidDeadlineTimeException();
+            }
+        }
+    }
+
+    private static boolean validDeadlineTime(String input) {
+        return input.substring(getTimePosition(input), getTimePosition(input) + 3).equals("/by");
+    }
+
+    private static boolean validEventTime(String input) {
+        return (input.substring(getTimePosition(input), getTimePosition(input) + 3).equals("/at"));
+    }
+
+
+    private static boolean isEmptyInput(String input, Command c) {
+        return switch (c) {
```

Comment:
This `return` block seems to be doing many things. You could consider simplifying this such that the `switch` statement is separate from the `return` statement. Hint: Create a new variable.